src/App.jsx

```diff
@@ -61,12 +63,23 @@ function App() {
     setTodoList([...filteredTodoList]);
   }
```

Comment:
Additional context to my suggestion below

```suggestion
  }
  
const ContentToDisplay = ({}) => (
      <>
      <p>{isLoading ? "Loading..." : "" }</p>
        <h1>Todo List</h1>
        <AddTodoForm onAddTodo={addTodo}/>
        <TodoList todoList={todoList} onRemoveTodo={removeTodo}/>
      </>
    )
```